mifosng-android/src/main/java/com/mifos/services/data/CenterPayload.java

```diff
@@ -4,18 +4,63 @@
  */
 package com.mifos.services.data;
 
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.mifos.api.local.MifosBaseModel;
+import com.mifos.api.local.MifosDatabase;
+import com.raizlabs.android.dbflow.annotation.Column;
+import com.raizlabs.android.dbflow.annotation.ModelContainer;
+import com.raizlabs.android.dbflow.annotation.PrimaryKey;
+import com.raizlabs.android.dbflow.annotation.Table;
+
 /**
  * Created by nellyk on 1/22/2016.
  */
-public class CenterPayload {
+@Table(database = MifosDatabase.class)
```

Comment:
@therajanmaurya When I was running the app, it crashed as you said earlier because previous database was already there. Then I found solution of increasing version number in 
mifosng-android/src/main/java/com/mifos/api/local/MifosDatabase.java
After that is was working fine. 
You can also see this comment - 
https://github.com/Raizlabs/DBFlow/issues/1059#issuecomment-258751594
I think that we don't need to add any migration code because we are not adding new column in the existing table. We are adding a completley new table. So increasing the version number should work as it worked in my case.
Please correct me if I am wrong.